packages/orbit-components/src/Alert/Alert.stories.tsx

```diff
@@ -161,9 +186,30 @@ InlineActions.story = {
   },
 };
 
-export const WithTextLink = () => {
-  const type = select("Type", Object.values(TYPE_OPTIONS), "info");
+InlineActions.args = {
+  type: TYPE_OPTIONS.INFO,
+  title: "You can change the title by changing the Title knob",
+  button: "I am a link",
+  closable: false,
+  icon: "Airplane",
```

Comment:
This seems to not be working :disappointed:. I see it's broken on master anyway. Can you take a look, please? (Same for playground)